requirements.txt

```diff
@@ -0,0 +1,14 @@
+Django>=1.5,<1.6
```

Comment:
Replace all of this with:

```
harvest
https://github.com/chop-dbhi/django-redcap/archive/master.zip
```

These two libraries will install all the things they need.